packages/keystone/fileAdapter/fileAdapter.js

```diff
@@ -5,10 +5,9 @@ const express = require('express')
 const { isEmpty, get } = require('lodash')
```

Comment:
It'll be better to add this package to "exports" section of the `package.json`. In case if we have plans to upgrade minimum requirement version of nodejs

Reply:
Done